pom.xml

```diff
@@ -109,6 +109,7 @@
                     <!-- TODO: set project ID. -->
                     <deploy.projectId>denniswillie-step-2020</deploy.projectId>
                     <deploy.version>1</deploy.version>
+                    <port>8181</port>
```

Comment:
I'll remove it. Sorry, it doesn't have to do anything with the project, but since I have started working on the remote desktop I cannot use the default 8080 port anymore, so each time I run a dev server I need to have the port set to something else.